spec/datetimeformat.html

```diff
@@ -991,7 +991,7 @@ <h1>FormatDateTimePattern ( _dateTimeFormat_, _patternParts_, _x_, _rangeFormatO
       </emu-note>
 
       <emu-note>
-        It is recommended that implementations use the time zone information of the IANA Time Zone Database.
+        A conforming implementation must recognize *"UTC"* and all other Zone and Link names (and <strong>only</strong> such names), and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.
```

Comment:
I'm not sure this note even belongs here, the above algorithm seems to be independent of the tzdb. My primary suggestion is to remove it, and my alternative secondary suggestion is to keep the reference.
```suggestion
        A conforming implementation must recognize *"UTC"* and all Zone and Link names of the IANA Time Zone Database (and <strong>only</strong> such names), and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.
```

Reply:
Agree. Removed in latest commit.